website/content/en/highlights/2021-08-24-vector-aggregator.md

```diff
@@ -1,32 +1,36 @@
 ---
 date: "2021-08-24"
-title: "Moving the aggregator chart to public beta"
-description: "The Vector aggregator Helm chart is now publicly available"
+title: "Moving the Aggregator Helm Chart to public beta"
```

Comment:
LOL, I lower cased these based on Luc's/Jesse's feedback

Reply:
Aggregator is a proper Vector role and should be capitalized. The same with Helm and Chart. Curious what @lucperkins thinks.

Reply:
https://github.com/timberio/vector/pull/8800#discussion_r692399148 previous discussion

Reply:
I'm of the opinion that we should keep it lower-cased until we are ready to make the change and then make the change site-wide.